src/index.html

```diff
@@ -6,7 +6,42 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__wrapper calendar__wrapper--month-length--31">
```

Comment:
Use BEM modifier syntax for 'month-length' like this: 'calendar__wrapper--month-length-31'.